cdap-master/src/main/java/co/cask/cdap/data/tools/flow/FlowQueuePendingCorrector.java

```diff
@@ -354,30 +351,13 @@ protected void configure() {
           bind(QueueClientFactory.class).to(HBaseQueueClientFactory.class).in(Singleton.class);
           bind(QueueAdmin.class).to(HBaseQueueAdmin.class).in(Singleton.class);
           bind(HBaseTableUtil.class).toProvider(HBaseTableUtilFactory.class);
-        }
-
-        @Provides
-        @Singleton
-        @Named("defaultStore")
-        @SuppressWarnings("unused")
-        public Store getStore(CConfiguration conf,
-                              LocationFactory locationFactory,
-                              NamespacedLocationFactory namespacedLocationFactory,
-                              final TransactionExecutorFactory txExecutorFactory,
-                              DatasetFramework framework,
-                              TransactionSystemClient txClient) {
-          return new DefaultStore(conf, locationFactory, namespacedLocationFactory,
-                                  txExecutorFactory, framework, txClient);
-        }
+          bind(Store.class).annotatedWith(Names.named("defaultStore")).to(DefaultStore.class).in(Singleton.class);
```

Comment:
The injector here is same as the one in ``HBaseQueueDebugger`` can we refactor it out as a module or something ?